delivery/src/main/java/com/storyous/delivery/common/DeliveryDetailFragment.kt

```diff
@@ -179,9 +179,10 @@ class DeliveryDetailFragment : Fragment() {
         order_note_group.visibility = View.GONE
     }
 
-    private fun updateCustomerInfo(customer: Customer) {
+    private fun updateCustomerInfo(customer: Customer, deskId: String?) {
         customer_detail_name.text = customer.name
         customer_detail_phone.text = customer.phoneNumber
         customer_detail_address.text = customer.deliveryAddress
+            ?: deskId?.let { getString(R.string.table, DeliveryConfiguration.translate(it)) }
```

Comment:
switch order of elvis, deskId has bigger priority than address. So when address would be set, deskId would be ignored.

Reply:
Proč to v přehledu objednávek bylo naopak?